### DISCUSSION TITLE
Add "album art" for radio stations.

### DISCUSSION THREAD

**Discussion by reddraconi** [navidrome/navidrome#2495] [Ideas]
It would be nice to have an option to upload a custom "album art" for radio stations that either do not have a favicon or
to override the favicon of a site that does have one configured. Maybe something in the 250x250px range?

Example 1:
The stream "Rektory" at "https://stream.nightride.fm/rektory.mp3" does not have an accompanying homepage
at https://nightride.fm/stations?stations=rektory. When streaming, Navidrome shows the default purple icon.

Example 2:
The streams "Chillsynth" and "Darksynth" at https://nightride.fom/stations?station=<stream_name> share a homepage, and thus, the same favicon. It would be nice to be able to override one (or both!) with a custom one.

Example 3:
The streams "Spacesynth" and "Beat Blender', hosted by Soma.FM have separate homepages, but share the same favicon.

**Comment by ScruffyDerHausmeister** [comment]
Maybe there could be a input field in the radio details for favicon url in order to download and use it in the player.

**Reply by jm355** [reply]
That's what symfonium does for it's radio implementation
